.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,29 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Title: Provide a general summary of your changes in the Title above -->
+<!--- Reviewers: Include @karstens, @GarthDB, @mrcjhicks, @lynnhao -->
 
 ## Description
 
-<!--- Describe your changes in detail -->
+<!--- Describe your changes in detail, including a list of changes -->
 
 ## Related Issue
 
 <!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If suggesting a new feature or change, please discuss it in #spectrum_tokens_talk or design workshop, first -->
 <!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
-
-## Motivation and Context
-
-<!--- Why is this change required? What problem does it solve? -->
-
-## How Has This Been Tested?
```

Comment:
From the designer's POV, I don't think there is testing required for the Tokens Studio changes, beyond what will be included in the process.